sdk/nodejs/__tests__/client.spec.ts

```diff
@@ -1,6 +1,5 @@
 import { getBucketingLib } from '../src/bucketing'
 import { DVCClient } from '../src/client'
-jest.mock('../src/bucketing')
```

Comment:
I'm guessing we want to revert this?

Reply:
it passes either way, I don't think it actually needs to be mocked here